public_html/about.php

```diff
@@ -94,16 +100,19 @@
         L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
             attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
         }).addTo(map);
-        
+
         locations.forEach(function(marker) {
             if (marker != null) {
                 latitude += marker.location[0];
                 longitude += marker.location[1];
-                L.marker(marker.location, {icon: greenIcon}).addTo(map).bindPopup(marker.full_name);
+                L.marker(marker.location, {icon: greenIcon}).addTo(map).bindPopup(
```

Comment:
Actually, the fix should be here if we are missing an image.

Reply:
I moved this markup into the PHP code so that it's handled in the same logic.